src/libs/E2E/tests/chatOpeningTest.e2e.ts

```diff
@@ -46,22 +45,6 @@ const test = (config: NativeConfig) => {
 
             console.debug(`[E2E] Entry: ${JSON.stringify(entry)}`);
 
-            if (entry.name === CONST.TIMING.CHAT_RENDER) {
-                E2EClient.submitTestResults({
```

Comment:
Why we don't measure first rendering time anymore?

I think it's quite important metric?..

Reply:
Discussed in DMs, we're unifying these 2 under the `open_report` event.

The workflow is now:
`sidebar_loaded` -> `trigger navigation event` -> `open_report` -> `collect the results`